app/Jobs/Network/SyncNetworkServerJob.php

```
@@ -0,0 +1,188 @@
+                if ($this->teardown) {
+                    $this->tearDown();
+
+                    return;
+                }
+
+                $this->syncToPresent();
+            });
+        });
+    }
+
+    public function failed(Exception $e): void
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $e. Configured minimum length is 3. (undefined)

(ShortVariable)